<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageGeneral</class>
 <widget class="QWidget" name="PageGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0" colspan="3">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelHudAltitude">
     <property name="text">
      <string>HUD Altitude Unit:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>comboBoxHudAltitude</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="ComboUnitsLength" name="comboBoxHudAltitude"/>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="ComboUnitsVelocity" name="comboBoxHudVelocity"/>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="Line" name="line_1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelHudVelocity">
     <property name="text">
      <string>HUD Velocity Unit:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>comboBoxHudVelocity</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButtonHudColor">
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelHudColor">
     <property name="text">
      <string>HUD Color:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>pushButtonHudColor</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>230</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QSlider" name="sliderHudOpacity">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelHudOpacity">
     <property name="text">
      <string>HUD Opacity:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>sliderHudOpacity</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frameHudColor">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QCheckBox" name="checkBoxWidescreen">
     <property name="text">
      <string>Widescreen dock areas window layout</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Sound Volume:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QSlider" name="sliderSoundVol">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboUnitsVelocity</class>
   <extends>QComboBox</extends>
   <header>gui/ComboUnitsVelocity.h</header>
  </customwidget>
  <customwidget>
   <class>ComboUnitsLength</class>
   <extends>QComboBox</extends>
   <header>gui/ComboUnitsLength.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonHudColor</tabstop>
  <tabstop>sliderHudOpacity</tabstop>
  <tabstop>comboBoxHudAltitude</tabstop>
  <tabstop>comboBoxHudVelocity</tabstop>
  <tabstop>checkBoxWidescreen</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
